packages/signature-controller/src/types.ts

```diff
@@ -7,12 +7,18 @@ export type OriginalRequest = {
   /** Unique ID to identify the client request. */
   id?: number;
 
+  /** Method of signature request */
+  method?: string;
+
   /** ID of the network client associated with the request. */
   networkClientId?: string;
 
   /** Source of the client request. */
   origin?: string;
 
+  /** Parameters in signature request */
+  params: string[];
```

Comment:
Are we sure these are always strings or should we use `unknown`?

Reply:
They are string AFAIK